en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
 gui.xaero_clear=Clear Set
 gui.xaero_from=From
 gui.xaero_to=To
-gui.xaero_used_seed=Seed To Use:
-gui.xaero_show_distance=Distance To WP
-gui.xaero_waypoints_scale=Ingame Waypoints Scale
-gui.xaero_block_colours=Block Colours
+gui.xaero_used_seed=Seed to Use:
+gui.xaero_show_distance=Distance to WP
+gui.xaero_waypoints_scale=In-Game Waypoint Scale
+gui.xaero_block_colours=Color Mode
 gui.xaero_accurate=Accurate
 gui.xaero_vanilla=Vanilla
 gui.xaero_lighting=Lighting
-gui.xaero_air_setting=Low Air Notification
-gui.xaero_air_low=Low Air
+gui.xaero_air_setting=Oxygen Notifications
+gui.xaero_air_low=Low Oxygen Value
 gui.xaero_looking_at=Looking At
-gui.xaero_all=All
-gui.xaero_display_redstone=Display Redstone
-gui.xaero_clear_set_message=Are you sure you would like to clear current set
-gui.xaero_clear_set_message2=All the waypoints will be deleted from the set.
+gui.xaero_all=All at Once
+gui.xaero_display_redstone=Show Redstone
+gui.xaero_clear_set_message=Are you sure you would like to clear the current set
+gui.xaero_clear_set_message2=All waypoints from this set will be deleted!
 gui.xaero_default=Default
 gui.xaero_create_set=New Waypoint Set
 gui.xaero_world=World
 gui.xaero_waypoint_set=Set
 gui.xaero_delete_set=Delete Set
-gui.xaero_delete_set_message=Are you sure you would like to delete current set
-gui.xaero_delete_set_message2=The set will be removed from current world.
-gui.xaero_toggle_waypoints=Toggle Ingame Waypoints
-gui.xaero_dots_scale=Entity Dots Scale
+gui.xaero_delete_set_message=Are you sure you would like to delete the current set
+gui.xaero_delete_set_message2=This set will be removed from the current world.
+gui.xaero_toggle_waypoints=Toggle In-Game Waypoints
+gui.xaero_dots_scale=Entity Dot Scale
 gui.xaero_entity_info=Entity Info
 gui.xaero_toggle_slime=Toggle Slime Chunks
 gui.xaero_toggle_grid=Toggle Chunk Grid
 gui.xaero_compass_over_wp=Compass Over Waypoints
 gui.xaero_display_hostile=Display Hostile Mobs
-gui.xaero_box_entity_info=An interface displaying information about the living entity that you are pointing at including players.
+gui.xaero_box_entity_info=An interface displaying information about the living entity that you are pointing at, including players.
 gui.xaero_entity_info_settings=Entity Info Settings
-gui.xaero_entity_info_stay=Keep Entity Info For
+gui.xaero_entity_info_stay=Continue Displaying For
 gui.xaero_entity_info_distance=Reach Distance
-gui.xaero_entity_info_max_hearts=Max Amount of Hearts
-gui.xaero_current_biome=Show Current Biome
+gui.xaero_entity_info_max_hearts=Max Heart Number
```

Comment:
Word "amount" if for uncountable nouns only. Hearts can be counted, and you even specify an exact value in the UI. "Max Number of Hearts" would be correct as well, but I decided to use the shorter variant for consistency reasons. But if you wish and like it more, we can use this schema: `max number of <noun>` in all places instead of: `max <noun_as_adj> number`.

Reply:
Oh, ok. What do you think about "Max Hearts"?

Reply:
It may be. But why do you prefer "Max Hearts" instead of "Max Heart Number"? Perhaps I should change "Entity Number" (`gui.xaero_entity_amount`) as well.

Reply:
Just because it was shorter. I think it's fine now though.